File: jdroid-android-glide/src/debug/java/com/jdroid/android/glide/debug/GlideDebugAppLifecycleCallback.java
package com.jdroid.android.glide.debug;

import android.content.Context;

import com.jdroid.android.debug.DebugSettingsHelper;
import com.jdroid.android.lifecycle.ApplicationLifecycleCallback;

public class GlideDebugAppLifecycleCallback extends ApplicationLifecycleCallback {

	@Override
	public void onProviderInit(Context context) {
		super.onProviderInit(context);

		DebugSettingsHelper.addPreferencesAppender(new GlideDebugPrefsAppender());
	}
}

File: jdroid-android-glide/src/debug/java/com/jdroid/android/glide/debug/GlideDebugAppLifecycleCallback.kt
package com.jdroid.android.glide.debug

import android.content.Context

import com.jdroid.android.debug.DebugSettingsHelper
import com.jdroid.android.lifecycle.ApplicationLifecycleCallback

class GlideDebugAppLifecycleCallback : ApplicationLifecycleCallback() {

    override fun onProviderInit(context: Context) {
        super.onProviderInit(context)

        DebugSettingsHelper.addPreferencesAppender(GlideDebugPrefsAppender())
    }
}
